diff --git a/android/build.gradle b/android/build.gradle
index b230e4d840ce752c9f7388bf7ff689073a58fa59..d71707324326215cde41cd91a38778ce53dc7de7 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -37,7 +37,7 @@ buildscript {
 
 // Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
+  archiveClassifier = 'sources'
   from android.sourceSets.main.java.srcDirs
 }
 
